src/app/analytics/dashboard-widget/dashboard-widget.component.scss

```
@@ -279,222 +279,236 @@
-    width: 40px;
-    height: 40px;
-    margin-left: -20px;
-    margin-top: -20px;
-    border-radius: 50%;
-    background: rgb(2 132 199 / 24%);
-    opacity: 0;
-    pointer-events: none;
-    transform: scale(0.4);
-    transform-origin: center;
-  }
+:host ::ng-deep {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)